proto/frequenz/api/dispatch/v1/dispatch.proto

```diff
@@ -80,6 +83,38 @@ service MicrogridDispatchService {
   rpc DeleteMicrogridDispatch(DeleteMicrogridDispatchRequest) returns (DeleteMicrogridDispatchResponse);
 }
 
+// Subscribe to a stream of microgrid dispatch requests.
+// This method provides real-time updates on newly or updated dispatch requests for edge-based
+// realtime decision making.
+message StreamMicrogridDispatchesRequest {
+  // ID of the microgrid to subscribe to
+  uint64 microgrid_id = 1;
+}
+
+// Response to a subscription request for a stream of microgrid dispatches.
+// Real-time information on dispatches affecting a certain microgrid are pushed through this response.
+message StreamMicrogridDispatchesResponse {
+  enum Type {
```

Comment:
I would call this `Event`, as we already have a `type` in the `Dispatch` object. It was a bit confusing at first.